packages/core/contracts/Escrow.sol

```diff
@@ -182,8 +180,21 @@ contract Escrow is IEscrow, ReentrancyGuard {
     }
 
     /**
-     * @dev Bulk payout workers
-     * Should fail if any of the transaction is failing.
+     * @dev Performs bulk payout to multiple workers
```

Comment:
This should stay @notice : Explain to an end user what this doe
@dev is the tag to :  Explain to a developer any extra details
@notice : Performs bulk payout to multiple workers 
@dev Escrow needs to be completed ... 

I think certik wanted to have a clear comments : 

- solc --devdoc --pretty-json Escrow.sol will return details with comments.

to check [https://coinsbench.com/natspec-the-right-way-to-comment-ethereum-smart-contracts-6762082252d4]